### PR TITLE
correct/tweak folder descriptions

### PR DESCRIPTION
**Pull request by bbolker**
Should be self-explanatory. I might have gotten carried away with minor language edits. I added "human", which I can't find an official description of, but I think my description is correct ... (we could confirm with Uwe Ligges or another CRAN maintainer if we wanted ...)

Addresses #29, #59

### CODE REVIEW

File: vignettes/dashboard.Rmd [hunk 1]
```diff
@@ -89,27 +89,22 @@ cran_incoming %>%
...
-- **newbies**: a specific queue for the manual inspection of first time CRAN submissions.
-- **pending**: the CRAN maintainers are waiting for an action on your side. You 
-should check your emails!
-- **waiting**: packages for which the CRAN team waits for an answer from the maintainer.
+in the review process. The exact meaning of each folder is detailed in articles from 
+the R Journal in [2018](https://journal.r-project.org/archive/2018-1/cran.pdf) and [2019](https://journal.r-project.org/archive/2019-1/cran.pdf), and updated by a [2019 mailing list post](https://stat.ethz.ch/pipermail/r-package-devel/2019q1/003631.html) (and [confirmed in 2022](https://stat.ethz.ch/pipermail/r-package-devel/2022q2/008084.html)):
+
+- **inspect**: your package is awaiting manual inspection by the CRAN team, probably because automated tests found a problem that is likely to be a false positive
+- **newbies**: a specific queue for the manual inspection of first-time CRAN submissions.
+- **pending**: a [CRAN] team member has to do a closer inspection and needs more time.
```
Comment (Bisaloo) [inline vignettes/dashboard.Rmd:97]: Why is CRAN between brackets here?

**Review by Bisaloo — COMMENTED** [review]
Looks good to me! I just added one minor review comment.

Thanks for your help! This has been a long-standing issue.

File: vignettes/dashboard.Rmd (hunk 1 above)
Comment (bbolker) [reply vignettes/dashboard.Rmd:97]: doesn't really need to be in brackets. The original statement is quoted from Uwe Ligges's e-mail to the list and didn't say "CRAN", but as I'm not pretending to directly quote (and I don't think Uwe will mind our nearly-plagiarizing this one sentence) I guess it's unnecessary/adds confusion.

**Review by Bisaloo — APPROVED** [review]

**Comment by Bisaloo** [comment]
Thanks!

**Pull request merged**